control/vehicle_cmd_gate/package.xml

```diff
@@ -33,10 +33,14 @@
   <depend>tier4_vehicle_msgs</depend>
   <depend>vehicle_info_util</depend>
```

Comment:
In my environment,
  <build_depend>rosidl_default_generators</build_depend>
is needed for ```ros2 topic echo /control/vehicle_cmd_gate/is_filter_activated```